ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule/Shrinking.hs

```diff
@@ -51,9 +51,9 @@ shrinkOtherPeers shrink Peers{honest, others} =
 -- trunk is unused, then it remains as an empty anchored fragment.
 trimBlockTree :: BlockTree TestBlock -> PointSchedule -> BlockTree TestBlock
 trimBlockTree bt ps =
-    let youngest = selectYoungest (pointScheduleBlocks ps)
-        trunk = keepOnlyAncestorsOf youngest (btTrunk bt)
-        branches = mapMaybe (fragmentToMaybe . keepOnlyAncestorsOf youngest . btbSuffix) (btBranches bt)
+    let leafs = blocksWithoutDescendents (pointScheduleBlocks ps)
```

Comment:
s/leafs/leaves/?

Reply:
:facepalm: thank you..

Reply:
Done in 018a85dc352b4dcfd31eddb878272e7a92faad86.